osu.Framework/Graphics/Rendering/IRenderer.cs

```diff
@@ -208,7 +208,7 @@ public interface IRenderer
         /// Flushes the currently active vertex batch.
         /// </summary>
         /// <param name="source">The source performing the flush, for profiling purposes.</param>
-        internal void FlushCurrentBatch(FlushBatchSource? source);
+        public void FlushCurrentBatch(FlushBatchSource? source);
```

Comment:
This one also seems to be unused in the tests?

So either

```suggestion
        internal void FlushCurrentBatch(FlushBatchSource? source);
```

to revert, or

```suggestion
        void FlushCurrentBatch(FlushBatchSource? source);
```

to remove the redundant modifier.

Reply:
Another drive-by change :eyes: